### PR TITLE
docs: add asset docs

### LINKED ISSUE #1430: Add documentation for assets
We should add a document describing what assets are in Rafiki:
- What is `assetScale` and `assetCode`
- How they can be managed
- Give some examples 
- Link to the liquidity document (#1429) for managing assets

### PR DESCRIPTION
**Pull request by sabineschaller**
## Changes proposed in this pull request

- adds some general info about assets as well as assets within Rafiki

## Context

- fixes #1430 

## Checklist

- [x] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [x] Documentation added
- [x] Make sure that all checks pass
- [ ] Postman collection updated

### CODE REVIEW

**Comment by sabineschaller** [comment]
I realize that there are some formatting issues with docusaurus. I hope starlight fixes that @huijing ?

**Comment by huijing** [comment]
> I realize that there are some formatting issues with docusaurus. I hope starlight fixes that @huijing ?

Yes, we are able to support MathJax `$$` syntax in Starlight.
Screenshot 2023-08-21 at 11 53 58 AM

File: packages/documentation/docs/concepts/accounting/assets.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Assets
```
Comment (mkurapov) [inline packages/documentation/docs/concepts/accounting/assets.md:1]: should we replace this: https://deploy-preview-1748--brilliant-pasca-3e80ec.netlify.app/docs/concepts/asset with what you wrote here?

File: packages/documentation/docs/concepts/accounting/assets.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+$$
+
+Hence, the above example represents $\frac{10000}{10^2} =100.00$ USD.
+
+Assets are represented in the `Asset` type due to JavaScript/Typescript's deficiencies when handling floats.
+
+## Assets in Rafiki
+
+When two Account Servicing Entities peer their Rafiki instances, they need to define the asset they are going to settle in. The Interledger packets they are going to exchange will then also be denominated in that asset.
```
Comment (mkurapov) [inline packages/documentation/docs/concepts/accounting/assets.md:30]:
```suggestion
When two Account Servicing Entities peer their Rafiki instances, they need to define the asset(s) they are going to settle in. The Interledger packets they are going to exchange will then also be denominated in that asset.
```

Comment (mkurapov) [reply packages/documentation/docs/concepts/accounting/assets.md:30]: Maybe we should also mention that you can only create payment pointers under a supported asset?

**Review by mkurapov — COMMENTED** [review]
Just need to update the links so the page builds

File: packages/documentation/docs/concepts/asset.md [hunk 3]
```diff
@@ -1,8 +1,34 @@
...
-- `withdrawalThreshold`: defines the minimum withdrawal amount
+The `Asset` type in Rafiki is comprised of a value, an asset code, and an asset scale.
+
+| Property   | Type    | Example |
+| ---------- | ------- | ------- |
+| value      | BigInt  | `10000` |
+| assetCode  | String  | `"USD"` |
+| assetScale | Integer | `2`     |
+
+The asset code SHOULD be an [ISO 4217 currency code](https://en.wikipedia.org/wiki/ISO_4217). The asset scale is the difference in orders of magnitude between the standard unit and a corresponding fractional unit. To convert from `Asset` to a currency amount that is more common to humans, apply the following formula:
```
[the remark this answers is not in the document]
Comment (sabineschaller) [reply packages/documentation/docs/concepts/asset.md:15]: Commodities are also part of ISO 4217 but I can mention that it should be used if it is available.

[the remark this answers is not in the document]
Comment (sabineschaller) [reply packages/documentation/docs/concepts/asset.md:15]: Also, this section explains why Ripple used **X**RP 😉

**Review by mkurapov — APPROVED** [review]

**Pull request merged**